### PR TITLE
fix standard dataset: adultdataset NotimplementError

### PR DESCRIPTION
**Pull request by autoih**
This PR fixes https://github.com/IBM/AIF360/issues/109 issue.

### CODE REVIEW

**Comment by autoih** [comment]
Hi @hoffmansc, can you help this PR?

**Comment by hoffmansc** [comment]
I think it would be simpler to just add `.to_numpy()` in these places. We would just have to bump up the requirement for pandas to `>=0.24.0`.

**Comment by autoih** [comment]
Thanks, @hoffmansc. Updated.

File: aif360/datasets/standard_dataset.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def __init__(self, df, label_name, favorable_classes,
                 unprivileged_values = list(set(df[attr]).difference(vals))
             else:
                 # find all instances which match any of the attribute values
-                priv = np.logical_or.reduce(np.equal.outer(vals, df[attr]))
+                priv = np.logical_or.reduce(np.equal.outer(vals, df[attr])).to_numpy()
```
Comment (hoffmansc) [inline aif360/datasets/standard_dataset.py]: this should be `...df[attr].to_numpy()))` actually

File: aif360/datasets/standard_dataset.py [hunk 2]
```diff
@@ -138,8 +138,8 @@ def __init__(self, df, label_name, favorable_classes,
             unfavorable_label = set(df[label_name]).difference(favorable_classes).pop()
         else:
             # find all instances which match any of the favorable classes
-            pos = np.logical_or.reduce(np.equal.outer(favorable_classes,
-                                                      df[label_name]))
+            pos = np.logical_or.reduce(np.equal.outer(favorable_classes, 
+                                                      df[label_name])).to_numpy()
```
Comment (hoffmansc) [inline aif360/datasets/standard_dataset.py]: same here: `df[label_name].to_numpy()))`

File: aif360/datasets/standard_dataset.py (hunk 1 above)
Comment (autoih) [reply aif360/datasets/standard_dataset.py]: Oh, yeah, too rush... Thanks, @hoffmansc.

**Comment by autoih** [comment]
Thanks @hoffmansc. Updated.

**Pull request merged**